packages/theming/theme-types/package.json

```diff
@@ -34,6 +34,7 @@
   "dependencies": {},
   "devDependencies": {
     "@types/react": "^16.9.34",
+    "@types/react-native": "^0.63.4",
     "react": "16.13.1",
     "react-native": "^0.63.4",
```

Comment:
nit: I feel like this is just a dependency now that we use ColorValue specifically..

Reply:
Latest commit addresses this.